sandy.vifm

```diff
@@ -0,0 +1,58 @@
+" sandy
+" Author: Nando, based on colorscheme snowhite by Jochen Schweizer
+" https://github.com/durcheinandr/vifm-colors
```

Comment:
What is this URL here for?

Reply:
Oops, sorry, that URL is not needed, it's part of the snowhite color scheme I used as a starting point.

Reply:
No problem. Please update your commit :)